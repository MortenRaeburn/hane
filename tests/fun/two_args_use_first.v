Definition foo : forall bar : Type{99}, (forall baz : Type{69}, Type{99}) := fun (bar : Type{99}) (baz : Type{69}) => bar.