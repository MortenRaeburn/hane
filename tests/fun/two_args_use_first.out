Definition foo : forall[Type{99}] (forall[Type{69}] (Type{99})) := fun[Type{99}] (fun[Type{69}] ('1)).
